packages/kie-bc-editors/src/GwtEditorWrapperFactory.ts

```diff
@@ -32,13 +32,19 @@ declare global {
   }
 }
 
+export const FACTORY_TYPE: string = "gwt";
```

Comment:
```suggestion
export const FACTORY_TYPE = "gwt";
```

That's what I meant :D

Reply:
Done.